modules/custom/d_demo/d_demo.install

```diff
@@ -1,16 +1,226 @@
+      'field_d_long_text' => [
+        'value' => '<ul><li>Lorem tortor</li><li>Fringilla sed</li><li>Vestibulum id</li><li>Eleifend</li><li>Varius in purus</li></ul>',
+        'format' => 'full_html',
+      ],
+    ],
+    '51c6b43b-1d38-41b8-a548-3651400be249' => [
+      'info' => 'Footer Menu Integer',
+      'type' => 'content_block',
+      'field_d_main_title' => 'Integer',
+      'field_d_long_text' => [
+        'value' => '<ul><li>Eleifend</li><li>Varius in purus</li><li>Integer</li></ul>',
+        'format' => 'full_html',
+      ],
+    ],
+    '387a89cf-03e8-4f0b-8805-a35c84acf7c7' => [
+      'info' => 'Footer Menu Quisque',
+      'type' => 'content_block',
+      'field_d_main_title' => 'Quisque',
+      'field_d_long_text' => [
+        'value' => '<ul><li>Lorem tortor</li><li>Fringilla sed</li><li>Vestibulum id</li><li>Eleifend</li><li>Varius in purus</li><li>Integer</li></ul>',
+        'format' => 'full_html',
+      ],
+    ],
+    '80b1af1f-d6c9-48a4-a248-f78bd16a2972' => [
+      'info' => 'Footer Menu Eleifend',
+      'type' => 'content_block',
+      'field_d_main_title' => 'Eleifend',
+      'field_d_long_text' => [
+        'value' => '<ul><li>Varius in purus</li><li>Integer</li></ul>',
+        'format' => 'full_html',
+      ],
+    ],
+  ];
+}
+
+/**
+ * Defines block placement in footer regions.
+ *
+ * @return array Block placement configuration
+ */
+function d_demo_footer_block_placement() {
+  return [
+    'footer_first' => [
+      'block_content:b9b4924d-e0d3-454f-95c0-b30b91001c51',
+      'block_content:012d4f2b-220f-4166-ab7a-84191dba9449',
+      'block_content:7f414b97-d117-41a8-8563-619355aa5772',
+    ],
+    'footer_second' => [
+      'system_menu_block:footer_example_menu',
+    ],
+    'footer_third' => [
+      'block_content:bc6831a2-adc3-46f4-b940-1ff1f9b9307f',
+      'block_content:51c6b43b-1d38-41b8-a548-3651400be249',
+    ],
+    'footer_fourth' => [
+      'block_content:387a89cf-03e8-4f0b-8805-a35c84acf7c7',
+      'block_content:80b1af1f-d6c9-48a4-a248-f78bd16a2972',
+    ],
+    'footer_fifth' => [
+      'block_content:bc6831a2-adc3-46f4-b940-1ff1f9b9307f',
+    ],
+    'footer_sixth' => [
+      'social_media_block'
+    ],
+  ];
+}
+
+/**
+ * Creates blocks and places them in configured regions.
+ *
+ * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+ * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+ * @throws \Drupal\Core\Entity\EntityStorageException
+ */
+function d_demo_create_footer_blocks() {
+  // Create custom block content.
+  foreach (d_demo_footer_block_content() as $uuid => $content) {
+    $storage = \Drupal::entityTypeManager()->getStorage('block_content');
+    $entity = $storage->create($content + ['uuid' => $uuid]);
+    $entity->save();
+  }
+
+  // Place blocks.
+  $i = 0;
+  foreach (d_demo_footer_block_placement() as $region => $blocks) {
+    foreach ($blocks as $block) {
+      Block::create([
```

Comment:
Create a function `d_content_init_move_block($plugin, $region, $weight = 0, $label = '')` in `d_content_init` module. If label is empty then set `label_display` to FALSE.